### ISSUE TITLE
exec-maven-plugin (exec:exec) output unexpected prefix for each line of stdout/stderr

### ISSUE THREAD

**Issue by gzm55** [apache/maven-mvnd#708]
```bash
$ mvnd -B -Dmvnd.noBuffering=true exec:exec -Dexec.executable=uname
[INFO] [stdout] Darwin

$ mvnd -B -Dmvnd.noBuffering=true exec:exec -Dexec.executable=java -Dexec.args=-version
[WARN] [stderr] openjdk version "11.0.2" 2019-01-15
[WARN] [stderr] OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
[WARN] [stderr] OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)

$ mvn -B -Dmvnd.noBuffering=true exec:exec -Dexec.executable=uname
Darwin
```

It is reproducible on mvnd 0.8.2. The output difference breaks the script to switch to mvnd.

**Comment by gzm55** [comment]
with `--raw-streams`, I've got the correct result. Can we add an env/property key for this cli switch?

**Comment by gzm55** [comment]
pr #712 is closed to prefer #719.

There is another behavior difference even with `--raw-streams` switch:

```
mvnd -N exec:exec -Dexec.executable=uname --raw-streams -l a.log
```

The output of `uname` is written to tty, not a.log, whereas the original `mvn` writes to a.log.

**Comment by gnodet** [comment]
This is also fixed by https://github.com/apache/maven-mvnd/commit/b2bd0aaae5ff8627d6425ede8c1de16e77b5f194
